Polygon_mesh_processing/examples/Polygon_mesh_processing/isotropic_remeshing_example.cpp

```diff
@@ -1,6 +1,7 @@
+#define EARLY
```

Comment:
To be removed?

Reply:
No longer needed. It was for an `#ifdef EARLY` which I had in the code.